samples/hyperparameter_tuning/my-hyperparameter-job.yaml

```diff
@@ -74,9 +74,9 @@ spec:
         s3DataSource:
           s3DataType: S3Prefix
           # The source of the validation data
-          s3URI: s3://<YOUR BUCKET/PATH>
+          s3URI: s3://<YOUR BUCKET/sagemaker/xgboost/validation
```

Comment:
same comment for `>`

Reply:
check all files for this.

Reply:
noted should only exist for these files changed as I updated the path will change